### PR TITLE
Presentation

### PR DESCRIPTION
**Pull request by seb879**
add presentation version

### CODE REVIEW

File: src/Components/ComponentPins/BottomPageCheckSvg.js [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+            d="M9.1,8.8v40.6c0,6.2,5.1,11.3,11.3,11.3h124c9.7-1.3,8.5-8.3,8.5-12.2c0-18.8,15.3-34.1,34.1-34.1
+                    c18.8,0,34.1,15.3,34.1,34.1c0,4-1.3,10.9,8.4,12.2h125.1c6.2,0,11.3-5.1,11.3-11.3V8.8H9.1z"
+          />
+        </g>
+      </g>
+    </g>
+    <g
+      onClick={() => dispatch({
+        type: 'NEXT_PAGE',
+        page: page + 1,
```
Comment (erabin05) [inline src/Components/ComponentPins/BottomPageCheckSvg.js]: ```page + 1``` must be in the reducer not in the action

File: src/Components/ComponentsCSS/ColorSvg.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+.st12{fill:none;stroke:#FFFFFF;stroke-width:0.25;stroke-miterlimit:10;}
+.st13{fill:none;stroke:#D02F5A;stroke-miterlimit:10;}
+.st14{clip-path:url(#SVGID_6_);fill:#FFFFFF;}
+.st15{enable-background:new;}
+.st16{clip-path:url(#SVGID_4_);}
+.st17{fill:none;stroke:#000000;stroke-width:0.25;stroke-miterlimit:10;}
+
+//BottomPageCloseSvg.js
+
+.st18{fill:#FFFFFF;stroke:#FFFFFF;stroke-width:0.25;;stroke-miterlimit:10;}
```
Comment (erabin05) [inline src/Components/ComponentsCSS/ColorSvg.scss]: double ```;```

File: src/Components/ComponentsCSS/ColorSvg.scss [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
```
Comment (erabin05) [inline src/Components/ComponentsCSS/ColorSvg.scss:114]: And did you forgot how to indent CSS ?

File: src/Components/ComponentsCSS/filterBar.scss [hunk 4]
```diff
@@ -1,47 +1,33 @@
...
+}
+
+.profile-logo{
+    width: 30%;
+    position: relative;
+    float: none;
+    z-index: 4;
+    float: right;
+    height : 8vh;
 }
```
Comment (erabin05) [inline src/Components/ComponentsCSS/filterBar.scss]: change to do in this file

File: src/Components/CreateNewPoi/PickCategoryFields.js [hunk 5]
```diff
@@ -12,19 +12,25 @@ const PickCategoryFiedls = ({
...
+      }}
       {...input}
     >
       <option value="">
             Choisissez une catégorie
       </option>
       {poiKeywordsDisplay.filter(keyword => keyword.importance === 2)
         .map(keyword => (
-          <option key={keyword.name} id={keyword.name} value={keyword.id} required>
+          <option key={keyword.name} id={keyword.name} value={keyword.id} onClick={() => dispatch({ type: 'SAVE_KEYWORD_NAME', selectedCategoryKeywordTwoName: keyword.name })} required>
```
Comment (erabin05) [inline src/Components/CreateNewPoi/PickCategoryFields.js]: to indent

File: src/Components/CreateNewPoi/TakeAPicture.js [hunk 6]
```diff
@@ -30,22 +31,38 @@ const uploadFileHandler = (e, formData) => {
...
+    <article style={{ display: 'grid', gap: '7vh' }}>
+      <div style={{ textAlign: 'center' }}>
+      Afin d'embellir votre expérience, veuillez fournir une photo.
+      </div>
+      <div>
+        <form
+          encType="multipart/form-data"
+          method="post"
+          onSubmit={e => uploadFileHandler(e, file)}
+          style={{
```
Comment (erabin05) [inline src/Components/CreateNewPoi/TakeAPicture.js]: Why CSS is in the JSX ?

File: src/Components/CreateNewPoi/TakeAPicture.js [hunk 7]
```diff
@@ -30,22 +31,38 @@ const uploadFileHandler = (e, formData) => {
...
+            display: 'grid',
+            gap: '10vh',
+          }}
+        >
+          <div style={{ display: 'grid' }} className="pictureUploadWrapper">
```
Comment (erabin05) [inline src/Components/CreateNewPoi/TakeAPicture.js]: the same

File: src/Components/PoiInformations.js [hunk 8]
```diff
@@ -15,21 +15,29 @@ const mapStateToProps = state => ({
...
         className="closePoiInformation"
         type="button"
         onClick={() => dispatch({ type: 'CLOSE_POI_INFOS', specificPoiInfos: [] })}
       />
 
-      <p className="poiName">{specificPoiInfos.name}</p>
+
+      <div
+        className="poiName"
+        onClick={() => dispatch({ type: 'TRANSITION_POI_INFOS', InformationPoiInfos: !InformationPoiInfos })}
```
Comment (erabin05) [inline src/Components/PoiInformations.js]: The change should be done in the reducer, not the action

File: src/Components/PoiInformations.js [hunk 9]
```diff
@@ -15,21 +15,29 @@ const mapStateToProps = state => ({
...
+        onKeyPress={() => dispatch({ type: 'TRANSITION_POI_INFOS', InformationPoiInfos: !InformationPoiInfos })}
```
Comment (erabin05) [inline src/Components/PoiInformations.js]: same

**Review by erabin05 — CHANGES_REQUESTED** [review]
Multiple files = multiple corrections
-> see in comments

File: src/Components/ComponentPins/BottomPageCheckSvg.js (hunk 1 above)
Comment (seb879) [reply src/Components/ComponentPins/BottomPageCheckSvg.js]: Done

File: src/Components/ComponentsCSS/ColorSvg.scss (hunk 2 above)
Comment (seb879) [reply src/Components/ComponentsCSS/ColorSvg.scss]: ok

File: src/Components/ComponentsCSS/ColorSvg.scss (hunk 3 above)
Comment (seb879) [reply src/Components/ComponentsCSS/ColorSvg.scss:114]: It's indent

File: src/Components/ComponentsCSS/filterBar.scss (hunk 4 above)
Comment (seb879) [reply src/Components/ComponentsCSS/filterBar.scss]: good

File: src/Components/CreateNewPoi/PickCategoryFields.js (hunk 5 above)
Comment (seb879) [reply src/Components/CreateNewPoi/PickCategoryFields.js]: ok

File: src/Components/CreateNewPoi/TakeAPicture.js (hunk 6 above)
Comment (seb879) [reply src/Components/CreateNewPoi/TakeAPicture.js]: Done

File: src/Components/CreateNewPoi/TakeAPicture.js (hunk 7 above)
Comment (seb879) [reply src/Components/CreateNewPoi/TakeAPicture.js]: corrected

File: src/Components/PoiInformations.js (hunk 8 above)
Comment (seb879) [reply src/Components/PoiInformations.js]: ok

File: src/Components/PoiInformations.js (hunk 9 above)
Comment (seb879) [reply src/Components/PoiInformations.js]: ok

**Review by erabin05 — APPROVED** [review]
Ok

**Pull request merged**